src/js/components/ArticleList.js

```diff
@@ -60,9 +60,64 @@ var PanelTitle = React.createClass({
   }
 });
 
+var FilterToggler = React.createClass({
+  render: function() {
+    return (
+      <div className="btn-group" role="group">{
+        this.props.choices.map((choice, i) => {
+          return <Button key={i} label={choice.label} size="sm"
+                         type={this.props.valueCheck === choice.value ? "info" : "default"}
+                         onClick={this.props.changeHandler(choice.value)} />;
+        })
+      }</div>
+    );
+  }
+});
+
+var Filters = React.createClass({
+  statics: {
+    unreadChoices: [
+      {label: "Read", value: false},
+      {label: "Unread", value: true}
+    ],
+    statusChoices: [
+      {label: "Default", value: "0"},
+      {label: "Archived", value: "1"},
+      {label: "Deleted", value: "2"},
```

Comment:
I don't think deleted articles are meant to be displayed in the UI. When fetching them, you should remove them from your client storage and forget them forever.

Otherwise it's more like a trash, and you need an additionnal deletion notion